spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -116,6 +116,16 @@ sealed class SpdxExpression {
      * ([ALLOW_CURRENT][Strictness.ALLOW_CURRENT]). Throws an [SpdxException] if validation fails.
      */
     abstract fun validate(strictness: Strictness)
+
+    /**
+     * Concatenate [this][SpdxExpression] and [other] using [SpdxOperator.AND].
+     */
+    infix fun and(other: SpdxExpression) = SpdxCompoundExpression(this, SpdxOperator.AND, other)
+
+    /**
+     * Concatenate [this][SpdxExpression] and [other] using [SpdxOperator.OR].
+     */
+    infix fun or(other: SpdxExpression) = SpdxCompoundExpression(this, SpdxOperator.OR, other)
```

Comment:
Just wondering, should we also add

    infix fun with(exception: SpdxLicenseExceptionExpression)

to `SpdxLicenseIdExpression` for completeness?

Reply:
Done.